### PR TITLE
Update on call docs to check needs triage for p0s and include information about upgrading demos

### PR DESCRIPTION
**Pull request by dkatzz**

### CODE REVIEW

File: docs/governance-and-management/project-management/on-call-guide/README.md [hunk 1]
```diff
@@ -50,13 +50,15 @@ Check if there are any current urgent bugs. If there are, make sure you know wha
...
    * [pac4j security announcement mailing list](https://groups.google.com/g/pac4j-security)
-4. Monitor staging deployments in the [#ci](https://app.slack.com/client/T01Q6PJQAES/C03UXPUEXU4) Slack channel. Investigate failed deployments and re-run if appropriate. (Note: our browser tests can be flakey and case deployments to fail. If this is the case, re-running the deployment will often fix the issue.)
-5. Check the [dependency dashboard](https://github.com/civiform/civiform/issues/2124) (once per shift)
+4. Check the [needs triage bugs](https://github.com/civiform/civiform/issues?q=is%3Aopen+is%3Aissue+label%3Aneeds-triage) that are created to make sure there aren't any P0s (daily)
+   * If there is something that looks critical, post in #eng-prod-incidents or post in #eng-general if you aren't sure.
+6. Monitor staging deployments in the [#ci](https://app.slack.com/client/T01Q6PJQAES/C03UXPUEXU4) Slack channel. Investigate failed deployments and re-run if appropriate. (Note: our browser tests can be flakey and case deployments to fail. If this is the case, re-running the deployment will often fix the issue.)
```
[the remark this answers is not in the document]
Comment (dkatzz) [reply docs/governance-and-management/project-management/on-call-guide/README.md:56]: I don't know why it showed up like that before, but I think it should be fixed now!

[the remark this answers is not in the document]
Comment (dkatzz) [reply docs/governance-and-management/project-management/on-call-guide/README.md:56]: Technically their deployments show up in the CI channel too, so I think its kind of included, but let me know if you think there's somewhere its worth calling out more.

File: docs/governance-and-management/project-management/on-call-guide/README.md [hunk 2]
```diff
@@ -87,3 +89,7 @@ CiviForm's dependencies are mostly listed in the [`build.sbt` file](https://gith
 CiviForm relies on [renovate bot](https://github.com/renovatebot/renovate) to automatically detect new versions of dependencies and create pull requests to update them. It is the on-call engineer's responsibility to review and merge these pull requests as they come in. **Do not simply approve and merge every pull request renovate bot creates.** While in most cases passing CI checks indicates the change is acceptable, that not always the case and more diligence is required ([here's an example](https://github.com/civiform/civiform/pull/2130#discussion\_r834714183)). Be sure you understand what is being updated before approving. If need be, get in touch with the broader engineering team to help evaluate a given PR. For PRs that break tests, add the "needs-triage" label to the PR so Exygy can prioritize fixing these issues.
 
 **Do not merge terraform-related dependency updates without first manually exercising the code, we do not have automated tests for terraform/deployment configuration.** Feel free to close related PRs and file issues for performing the upgrade.
+
+#### Upgrading demo sites
```
[the remark this answers is not in the document]
Comment (dkatzz) [reply docs/governance-and-management/project-management/on-call-guide/README.md:94]: Yeah that's a good call! Updated! And also sent an email ensuring nobody has a problem with us doing that.

File: docs/governance-and-management/project-management/on-call-guide/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gwendolyngoetz) [reply docs/governance-and-management/project-management/on-call-guide/README.md:56]: If we want to exclude it I can remove it from posting into the CI channel. I don't care either way about being pinged by people, but no one needs to worry about Seattle's staging site at this micro level. We don't offer it to anyone else.

[the remark this answers is not in the document]
Comment (dkatzz) [reply docs/governance-and-management/project-management/on-call-guide/README.md:56]: Up to you! Its kind of nice I think to see it and confirm that its on the same version as the normal staging site when asking Matt for testing, but I'm somewhat indifferent. I'll resolve this comment for now and we can chat more about it in slack or a team meeting.

**Pull request merged**